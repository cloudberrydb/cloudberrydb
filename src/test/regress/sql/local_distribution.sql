-- Creates db on coordinator shard without distributing

CREATE TABLE local_table (asd Int) DISTRIBUTED LOCAL;

EXPLAIN SELECT * FROM local_table;
