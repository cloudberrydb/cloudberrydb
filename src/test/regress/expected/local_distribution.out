-- Creates db on coordinator shard without distributing
CREATE TABLE local_table (asd Int) DISTRIBUTED LOCAL;
EXPLAIN SELECT * FROM local_table;
                           QUERY PLAN                           
----------------------------------------------------------------
 Seq Scan on local_table  (cost=0.00..106.30 rows=9630 width=4)
 Optimizer: Postgres query optimizer
(2 rows)

